IFC4x3/Properties/p/PoissonRatio_07KCzf3UjDcOJp9bdH8Dbb/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PoissonRatio_07KCzf3UjDcOJp9bdH8Dbb" Name="PoissonRatio" UniqueId="0750cf69-0deb-4d99-84f3-2659d120d965" PrimaryDataType="IfcPositiveRatioMeasure" />
```

Comment:
Reuse existing.

Reply:
See also [BookB-Pset_GeoScienceFeatureTypeIntactRock](https://github.com/bSI-InfraRoom/IFC-Specification/pull/627#top).
There is one "PoisonsRatio" used in Pset_SolidStratumCapacity which feels named wrongly but with a generic enough definition: "Ratio of transverse contraction strain to longitudinal extension strain in the direction of stretching force.". Then there is one "PoissonRatio" with the definition "A measure of the lateral deformations in the elastic range." which feels a bit vague to me. Can we generalize all these into one?
The one used here has the definition "Ratio between radial/horizontal strain and axial strain of a (geo)material when compressed."

Reply:
Fixed in [b5a0c83](https://github.com/bSI-InfraRoom/IFC-Specification/pull/630/commits/b5a0c83eb99dfdf079584512ed47156833ffba45)